src/components/views/page-title.ts

```diff
@@ -0,0 +1,8 @@
+import tw from 'tailwind-styled-components';
```

Comment:
instead of having page-title as its own, it should be part of `views.ts` which will have multiple exports - and its `PageTitle` not `WelcomeTitle` - see the MyBucks project for more

Reply:
I understand but I only did it this way as a demo since we still have to modify a lot of these files for the demo project in the end.